virtwho/virt/fakevirt/fakevirt.py

```diff
@@ -1,10 +1,39 @@
 
 import json
 
+from virtwho.config import VirtConfigSection, accessible_file
 from virtwho.virt import Virt, VirtError, Guest, Hypervisor
 from virtwho.util import decode
 
 
+class FakeVirtConfigSection(VirtConfigSection):
+    VIRT_TYPE = 'fake'
+
+    def __init__(self, *args, **kwargs):
+        super(FakeVirtConfigSection, self).__init__(*args, **kwargs)
+
+        self.add_key('file', validation_method=self._validate_fake_virt_file, required=True)
+        self.add_key('is_hypervisor', validation_method=self._validate_str_to_bool, default=True,
+                     required=True)
+
+    def _validate_fake_virt_file(self, key):
+        result = None
+        file_path = self.get(key)
+        # Ensure the value we've got is a file we can access
+        try:
+            accessible_file(file_path)
+            is_hypervisor = self.get('is_hypervisor')
+            if is_hypervisor is True:
+                # Try to read and parse the file as if it were a host guest mapping
+                FakeVirt.read_host_guest_mapping_from_file(file_path)
+            elif is_hypervisor is False:
+                # Try to read and parse the file as if it were a domain list
+                FakeVirt.list_domains_from_file(file_path)
+        except (VirtError, ValueError) as e:
```

Comment:
There is IMHO too much code between `try-except`. I guess `ValueError` can come only from `accessible_file(file_path)` and `VirtError` can come only from methods of `FakeVirt`. I would split these two cases into two `try-excepts`.

Reply:
I think it is more concise as is but will make the changes so we can compare.